modin/test/test_headers.py

```diff
@@ -44,6 +44,8 @@ def test_line_endings():
         if any(i in subdir for i in [".git", ".idea", "__pycache__"]):
             continue
         for file in files:
+            if file.endswith(".parquet"):
```

Comment:
In binary format there is no need to check these characters (`\r\n`).

Reply:
This sequence of characters may have a different meaning for binary formats. If, for example, you try to replace `\r\n` with `\n`, the arrow will give a message that parquet file is corrupted.